.mergify.yml

```diff
@@ -3,13 +3,15 @@ shared:
       {{title}} (#{{number}})
       
       {{body | get_section("## What's changed and what's your intention?")}}
-      
       {% for user in approved_reviews_by %}
       Approved-By: {{user}}
```

Comment:
Can we add an `@` before the username?

Reply:
Will the `@` lead to noisy notification? 🥵

Just tested, did you receive any notification?

<img width="751" alt="image" src="https://user-images.githubusercontent.com/37948597/208644580-44a0b393-ccab-4525-a354-ab5af3378b21.png">

Reply:
Good catch. 🤣

|
<img width="451" alt="image" src="https://user-images.githubusercontent.com/25862682/208645338-666a1b16-0e41-4d0f-b3f7-dfc1ec6d2043.png">